code/go/0chain.net/chaincore/chain/state_handler.go

```diff
@@ -227,7 +227,9 @@ func (c *Chain) GetSCRestPoints(w http.ResponseWriter, r *http.Request) {
 
 	sort.Strings(names)
 	for _, funcName := range names {
-		friendlyName := strings.TrimLeft(funcName, "/")
+		// friendlyName := strings.TrimLeft(funcName, "/")
+		paths := strings.Split(funcName, "/")
+		friendlyName := "/" + paths[len(paths)-1]
 		fmt.Fprintf(w, `<tr><td>%v</td><td><li><a href='%v'>%v</a></li></td></tr>`, friendlyName, key+funcName, "/v1/screst/*"+funcName+"*")
```

Comment:
It is the second param, i.e. `key+funcName` that is incorrect, please build the image, run and check the links

Reply:
@peterlimg  fixed this. Also testing this by deploying this on a network.
https://helm.0chain.net/sharder01/v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7